### PR TITLE
startTime, imageQuality ++

### PR DESCRIPTION
**Pull request by magnurud**

### CODE REVIEW

File: Lucidtech/Las/Client.cs [hunk 1]
```diff
@@ -364,12 +364,14 @@ public object CreateBatch(string? name = null, string? description = null)
             string documentId,
             string modelId,
             int? maxPages = null,
-            bool? autoRotate = null
+            bool? autoRotate = null,
+            string? imageQuality = null
```
Comment (augustak) [inline Lucidtech/Las/Client.cs:368]: Not Enum?

File: Lucidtech/Las/Client.cs [hunk 2]
```diff
@@ -806,13 +809,15 @@ public object CreateBatch(string? name = null, string? description = null)
...
+        /// <param name="startTime"> Utc start time that will replace the original start time of the execution</param>
         /// <returns>Transition execution response from REST API</returns>
         public object UpdateTransitionExecution(
             string transitionId,
             string executionId,
             string status,
             Dictionary<string, string>? output = null,
-            Dictionary<string, string>? error = null
+            Dictionary<string, string>? error = null,
+            string? startTime = null
```
Comment (augustak) [inline Lucidtech/Las/Client.cs:820]: Is there a better type akin to python datetime we can use here?

File: Lucidtech/Las/Client.cs [hunk 3]
```diff
@@ -826,10 +831,32 @@ public object CreateBatch(string? name = null, string? description = null)
...
+
+            if (startTime != null) {
+                body.Add("startTime", startTime);
+            }
+
             var request = ClientRestRequest(Method.PATCH, url, body);
             return ExecuteRequestResilient(RestSharpClient, request);
         }
 
+        /// <summary>Send heartbeat for a manual execution,
```
Comment (augustak) [inline Lucidtech/Las/Client.cs:843]: "Send heartbeat for a manual execution to signal that we are still working on it. Must be done at minimum once every 60 seconds or the transition execution will time out"

File: Lucidtech/Las/Client.cs [hunk 4]
```diff
@@ -1268,6 +1295,10 @@ private static bool FatalCode(HttpStatusCode code)
         
         private object JsonDecode(IRestResponse response)
         {
+            if (response.StatusCode == HttpStatusCode.NoContent)
+            {
+                return null;
+            }
             if (response.StatusCode == HttpStatusCode.Forbidden)
```
Comment (augustak) [inline Lucidtech/Las/Client.cs:1302]: else if? (or switch)

File: Lucidtech/Las/Client.cs [hunk 5]
```diff
@@ -1268,6 +1295,10 @@ private static bool FatalCode(HttpStatusCode code)
         
         private object JsonDecode(IRestResponse response)
         {
+            if (response.StatusCode == HttpStatusCode.NoContent)
+            {
+                return null;
```
Comment (augustak) [inline Lucidtech/Las/Client.cs:1300]: In Java SDK we are returning a client constructed response, should we not do the same here?

File: Lucidtech/Las/Client.cs (hunk 1 above)
Comment (magnurud) [reply Lucidtech/Las/Client.cs:368]: We don't have enums for the other "enum-like" variables like transitionType and order, so i suggest we take that in another PR for consistency

**Review by augustak — APPROVED** [review]

**Pull request merged**